plugins/inputs/interrupts/interrupts.go

```diff
@@ -143,6 +205,18 @@ func reportMetrics(measurement string, irqs []IRQ, acc telegraf.Accumulator, cpu
 	}
 }
 
+func reportSpuriousMetrics(irqs []IRQ, acc telegraf.Accumulator) {
+	for _, irq := range irqs {
+		tags, _ := gatherTagsFields(irq)
+		spuriousFields := map[string]interface{}{
+			"count":     irq.SpuriousCount,
+			"unhandled": irq.SpuriousUnhandled,
+			"total":     irq.Total,
+		}
+		acc.AddFields("spurious_interrupts", spuriousFields, tags)
+	}
+}
+
```

Comment:
Can you please fold that into `reportMetrics` to reduce the amount of duplicate code.
Furthermore, why not just add the "spurious" fields to the usual metric?